addon-test-support/focus.js

```diff
@@ -19,14 +19,23 @@ export function focus(selector) {
     let type = el.type;
     if (type !== 'checkbox' && type !== 'radio' && type !== 'hidden') {
```

Comment:
I've just noticed this, that we can't focus on checkbox or radio. That's interesting, as I was not able to find the reason, that we can't focus on them. Can someone point out, what is the reason for that?